wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientTableViewCell.swift

```diff
@@ -203,7 +157,10 @@ class ClientTableViewCell: UITableViewCell, DynamicTypeCapable {
     }
 
     func redrawFont() {
-        updateFingerprint()
         updateLabel()
     }
+
+    func showOrHideMLS() {
+        mlsThumbprintLabel.heightAnchor.constraint(equalToConstant: 0).isActive = userClient?.mlsPublicKeys.ed25519 == nil
```

Comment:
**question:** don't we need to do layoutIfNeeded after the constraint change?

Reply:
But I could add it just in case